src/common/crypto/platform/openssl/crypto.cpp

```diff
@@ -281,6 +282,11 @@ ExpectedPrivateKey LoadFrom(const Args &args) {
 #endif // ndef MENDER_CRYPTO_OPENSSL_LEGACY
 
 ExpectedPrivateKey PrivateKey::Load(const Args &args) {
+	// Load OpenSSL config
+	if ((CONF_modules_load_file(NULL, NULL, 0) != 1)) {
```

Comment:
Use `OPENSSL_SUCCESS` instead of hard-coded 1
```suggestion
	if ((CONF_modules_load_file(NULL, NULL, 0) != OPENSSL_SUCCESS)) {
```